warehouse/slave/worker.go

```diff
@@ -508,7 +512,7 @@ func (sw *slaveWorker) runAsyncJob(ctx context.Context, asyncjob jobs.AsyncJobPa
 	return result, nil
 }
 
-func (sw *slaveWorker) destinationFromSlaveConnectionMap(destinationId, sourceId string) (model.Warehouse, error) {
+func (sw *worker) destinationFromSlaveConnectionMap(destinationId, sourceId string) (model.Warehouse, error) {
```

Comment:
change sw to w?

Reply:
OOps missed updating the receiver. Updating it.